lib/tyk/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+services:
+  tyk:
+    container_name: tyk
```

Comment:
container name should not be needed

Reply:
how would services communicate without a name to resolve to?